README.md

```diff
@@ -24,7 +24,7 @@ Setup
 -------
 
 1. Copy `config.conf.example` to `config.conf` and edit it.
-1. Then type `make`.
+2. Then type `make`.
```

Comment:
Is there a reason you changed this?

Reply:
See commit dfe098bc8d54dcf252a4fc742e3bfc3f659faaa1 but I removed the other line in commit 7c9f41a43e14b5ca6e9f619b2a6c190ff5c7ac19

Reply:
OK, let's revert it then?

Reply:
Why, shouldn't it be 1. and 2. instead of 1. and 1. ?

Reply:
The latter never requires changing irrelevant lines for inserting an item,
i.e. smaller diffs.